hs_tracking/models.py

```diff
@@ -225,6 +234,34 @@ def recent_resources(cls, user, n_resources=5, days=60):
             .filter(variable__timestamp=F('last_accessed'))\
             .order_by('-last_accessed')[:n_resources]
 
+    @classmethod
+    def popular_resources(cls, n_resources=5, days=60, today=None):
+        """
+        fetch the most recent n resources with which a specific user has interacted
+
+        :param n_resources: the number of resources to return.
+        :param days: the number of days to scan.
+
+        The reason for the parameter `days` is that the runtime of this method
+        is very dependent upon the days that one scans. Thus, there is a tradeoff
+        between reporting history and timely responsiveness of the dashboard.
+        """
+        # TODO: document actions like labeling and commenting (currently these are 'visit's)
+        if today is None:
+            today = datetime.now()
+        return BaseResource.objects.filter(
```

Comment:
Are timezones a consideration here?

Reply:
My documentation says that now() is timezone-aware.